BullsAndCowsRunner/Program.cs

```diff
@@ -9,14 +10,42 @@ public static void Main(string[] args)
         {
             SecretGenerator secretGenerator = new SecretGenerator();
             BullsAndCowsGame game = new BullsAndCowsGame(secretGenerator);
+
+            const int maxCount = 6;
+            var count = 0;
             while (game.CanContinue)
```

Comment:
why is cancontinue here? i think it is always true

Reply:
there is a " break" in the following